behat.yml

```diff
@@ -0,0 +1,34 @@
+# Run admin behat tests with this command (installed with silverstripe/installer)
+# Note that admin behat tests require CMS module
+# ========================================================================= #
+# vendor/bin/selenium-server-standalone -Dwebdriver.firefox.bin="/Applications/Firefox31.app/Contents/MacOS/firefox-bin"
+# vendor/bin/serve --bootstrap-file vendor/silverstripe/admin/tests/behat/serve-bootstrap.php
+# vendor/bin/behat @admin
+# ========================================================================= #
+default:
+  suites:
+    admin:
+      paths:
+        - %paths.modules.admin%/tests/behat/features
+      contexts:
+        - SilverStripe\Framework\Tests\Behaviour\FeatureContext
+        - SilverStripe\Framework\Tests\Behaviour\CmsFormsContext
+        - SilverStripe\Framework\Tests\Behaviour\CmsUiContext
+        - SilverStripe\BehatExtension\Context\BasicContext
+        - SilverStripe\BehatExtension\Context\EmailContext
+        - SilverStripe\BehatExtension\Context\LoginContext
+        -
+          SilverStripe\BehatExtension\Context\FixtureContext:
+            - %paths.modules.admin%/tests/behat/features/files/
+
+  extensions:
+    SilverStripe\BehatExtension\MinkExtension:
+      default_session: facebook_web_driver
+      javascript_session: facebook_web_driver
+      facebook_web_driver:
+        browser: chrome
+        wd_host: "http://127.0.0.1:9515" #chromedriver port
+
+    SilverStripe\BehatExtension\Extension:
+      screenshot_path: %paths.base%/artifacts/screenshots
+      bootstrap_file: vendor/silverstripe/framework/tests/behat/serve-bootstrap.php
```

Comment:
This file has been removed in the corresponding PR - do we need to move it to this module instead?